app/features/conference/components/Conference.js

```diff
@@ -265,10 +266,13 @@ class Conference extends Component<Props, State> {
         new RemoteControl(iframe); // eslint-disable-line no-new
         setupAlwaysOnTopRender(this._api);
         setupWiFiStats(iframe);
+        setupPowerMonitorRender(iframe);
 
+        this._api.on('suspendDetected', (event: Event) => {
```

Comment:
Maybe add `this. _onVideoConferenceEnded` as a value for the listener? You may need to do:`this. _onVideoConferenceEnded = this._onVideoConferenceEnded.bind(this)` in the constructor but I'm not sure.